md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/BatchAPI.java

```diff
@@ -234,11 +226,11 @@ RestWrapper insert(@ModelAttribute("batch")
 
             Long autoGenBatchId = batchDAO.insert(jpaBatch);
             batch.setBatchId(autoGenBatchId);
-            //Batch batches = s.selectOne("call_procedures.InsertBatch", batch);
 
             restWrapper = new RestWrapper(batch, RestWrapper.OK);
-            LOGGER.info("Record with ID:" + batch.getBatchId() + " inserted in Batch by User:" + principal.getName() + batch);
+            LOGGER.info(RECORDWITHID + batch.getBatchId() + " inserted in Batch by User:" + principal.getName() + batch);
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)